mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/id/objectid/Bar.java

```diff
@@ -18,6 +18,7 @@
 import javax.persistence.OneToMany;
 
 import org.bson.types.ObjectId;
+import org.jboss.logging.processor.util.Objects;
```

Comment:
I don't think this is the one you want to use :)

Reply:
Damn! :)
Thanks

Reply:
Add it to the checkstyle blacklist? We did similar things for e.g. to ban the deprecated junit helpers.